docs/source/tutorials/finetuning.md

```diff
@@ -0,0 +1,98 @@
+# Tutorial : Finetuning from pretrained models
```

Comment:
If possible, we should create simple colabs for all these so that users can easily follow.